sbnanaobj/StandardRecord/SRHeader.cxx

```diff
@@ -23,7 +23,9 @@ namespace caf
   ngenevt(0),
   pot(0.),
   mctype(caf::kMCUnknown),
-  det(std::numeric_limits<Det_t>::signaling_NaN())
+  det(std::numeric_limits<Det_t>::signaling_NaN()),
```

Comment:
I don't think this makes sense. I wish std::numeric_limits would refuse to compile for integer types that don't actually have a NaN. Here I would use `-1`.

Reply:
This PR does not have to do with that line of code, but I can change it.

Reply:
I guess you added a comma to it :smile: